core/src/main/java/nl/nn/adapterframework/senders/JavascriptSender.java

```diff
@@ -68,7 +69,7 @@ public class JavascriptSender extends SenderSeries {
 	private String fileInput;
 
 	public enum JavaScriptEngines {
-		J2V8(J2V8.class), NASHORN(Nashorn.class), RHINO(Rhino.class);
+		J2V8(J2V8.class), @Deprecated NASHORN(Nashorn.class), @Deprecated RHINO(Rhino.class);
```

Comment:
Can you add `@ConfigurationWarning` on the deprecated elements too? Or will that not be displayed in the ConfigurationWarnings anyway?

Reply:
I'll add it anyways.. is faster than checking if it is being used. We'll remove it in 8.0 anyways.